utils/src/ooinstall/openshift_ansible.py

```diff
@@ -153,17 +156,17 @@ def write_inventory_vars(base_inventory, multiple_masters, lb):
 def write_proxy_settings(base_inventory):
     try:
         base_inventory.write("openshift_http_proxy={}\n".format(
-                                                            CFG.settings['openshift_http_proxy']))
+            CFG.settings['openshift_http_proxy']))
```

Comment:
Do you want to apply that to lines 134 and 137 as well, those where the lines that I was comparing to from the original comment.